converter/ena/base.py

```diff
@@ -33,6 +33,9 @@ def __init_is_update(self):
     def init_ena_set(self):
         self.ena_set = etree.XML(f'<{self.ena_type.upper()}_SET />')
 
+    def init_xml_spec(self):
```

Comment:
you could add abstractmethod decorator for all your abstract methods  so you're sure all classes extending it would implement those methods https://stackoverflow.com/questions/7196376/python-abstractmethod-decorator

Reply:
Thanks, I am going to use it in my next changes.